tests/test_actions/test_notification.py

```diff
@@ -0,0 +1,12 @@
+"""test module."""
+try:  # py3
+    from unittest import mock
+except ImportError:  # py2
+    import mock
+
+
+def test_push():
+    info = mock.Mock()
+    with mock.patch('melissa.profile_loader.load_profile'):
```

Comment:
this is a correct start by patching `load_profile` on `profile_loader` module.

but before that declare `dict` variable contain `push_bullet` and `va_name` keys. the value of those keys will not be a problem. this variable will be used as mocked profile

after that mock `Pushbullet` on `notification` module. the path is `melissa.actions.notification.Pushbullet`.

in comparison with `spelling` module,

        with mock.patch('melissa.actions.spelling.tts') as m_tts:

you can decide the name of mocked `Pushbullet` object, but add `m_` for uniformity with other test.

after that overwrite the `profile` on `notification` module. e.g. the declared dict variable is `m_profile`:

    notification.profile.data = m_profile  # put this before run notification.push

the mocked pushbullet now can be tested after you run `notification.push` (the line is already correct). after that check it with [assert_has_calls](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_has_calls)

if you want to know the calls for mocked `pushbullet object. you can put pdb's `set_trace` after `notification.push` and print [mock_calls](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.mock_calls) of mocked `pushbullet` obj.